prow/github/client.go

```diff
@@ -70,7 +70,7 @@ type Client struct {
 	email   string
 }
 
-const (
+var (
```

Comment:
why is this now var?

Reply:
So that the values can be modified while testing.

Reply:
perhaps in a future update these can be scoped to a struct and associated with the client? the package could have a default client similar to `net/http`